src/config/app.ts

```diff
@@ -26,6 +26,7 @@ try {
 export const yamlConfig = _.merge(defaultConfig, customConfig)
 
 export const MEMO_SHARING_SATS_THRESHOLD = yamlConfig.limits.memoSharingSatsThreshold
+export const ONCHAIN_MIN_CONFIRMATIONS = yamlConfig.onChainWallet.minConfirmations || 2
```

Comment:
no need to override the default here. the yaml is assumed to always be populated (unless this has been changed recently @vindard ?)

Reply:
I thought the same but didn't want to take risks with this, please let me know

Reply:
No changes from my end with that expected behavior

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/428/commits/0adda6365e5311a71c03a054caad4adf90046630